frontend/ui/widget/inputdialog.lua

```diff
@@ -478,13 +481,18 @@ function InputDialog:getAddedWidgetAvailableWidth()
     return self._input_widget.width
 end
 
--- Close the keyboard if we tap anywhere outside of the keyboard (that isn't an input field, where it would be caught via InputText:onTapTextBox)
+-- Tap outside of inputbox to hide the keyboard (inside the inputbox it is caught via InputText:onTapTextBox).
```

Comment:
I don't recall what happens if we tap inside the input *dialog*, and not an input *field*. (I would assume nothing? Which is why the wording was so careful ;p).

Reply:
The keyboard becomes hidden, on the second tap - the dialog would be closed.
The reactions on tap inside the dialog (not inputfield and buttons) and outside of it are the same.

Reply:
>  on the second tap - the dialog would be closed.
> The reactions on tap inside the dialog (not inputfield and buttons) and outside of it are the same.

Mhhh, this doesn't feel right to me.
If we tap outside, it is fine to close the widget.
But nowhere/never we have "tap anywhere inside a widget close the widget" (except on an obvious Close or x button).
I just tested it, and it indeed feels super odd and unexpected UX.
(OK, I don't usually tap much on a widget - but I can imagine one doing that resting his fingers, seeing the keyboard disappear, tapping again at the same place to make it reappear, and poof, lost the input text).

Reply:
We *should* be able to hitcheck that somewhere, though? (without checking anything: `self[1].dimen`, maybe?)

Reply:
I was late to the party I see ;o).